settings.py

```diff
@@ -8,7 +12,11 @@
 ]
 
 import aldryn_addons.settings
-aldryn_addons.settings.load(locals())
+
+aldryn_addons.settings.load(
```

Comment:
It's ok to have the ``debug`` argument as an optional way to override the behaviour. Please leave the signature unchanged here (just ``aldryn_addons.settings.load(locals())``).

And set ``debug=boolean_ish(os.environ.get('ALDRYN_ADDONS_DEBUG', False))`` as the defualt if debug was not passed in explicitly.